common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -25,10 +26,29 @@
 # OBSOLETE: This obsoletes 'type'
 CLASS_PRIORITY = ['video', 'problem']
 
+# Make '_' a no-op so we can scrape strings. Using lambda instead of
+#  `django.utils.translation.ugettext_noop` because Django cannot be imported in this file
+_ = lambda text: text
+
+
+class VerticalFields(object):
+
+    discussions_enabled = Boolean(
+        display_name=_("Discussions Enabled"),
+        help=_(
+            "This setting enable discussions for this course unit. A discussion UI will show up under this Unit."
+        ),
+        default=False,
+        scope=Scope.settings,
+    )
+
 
 @XBlock.needs('user', 'bookmarks')
 @XBlock.wants('completion')
-class VerticalBlock(SequenceFields, XModuleFields, StudioEditableBlock, XmlParserMixin, MakoTemplateBlockBase, XBlock):
+class VerticalBlock(
+    SequenceFields, XModuleFields, StudioEditableBlock,
+    XmlParserMixin, MakoTemplateBlockBase, XBlock, VerticalFields
```

Comment:
VerticalFields should be the first one here, and XBlock should be the last.

Reply:
Thanks for the input! I'll fix this.